src/app/components/progress-table/progress-table.component.html

```diff
@@ -170,14 +171,39 @@
         </ion-button>
       </ng-template>
     </ngx-datatable-column>
-  </ngx-datatable>
 
-  <ion-button color="primary" fill="clear" (click)="switchType()">
-    <ng-container *ngIf="type === 'team'">
-      VIEW PROGRESS BY STUDENT
-    </ng-container>
-    <ng-container *ngIf="type === 'student'">
-      VIEW PROGRESS BY PROJECT
-    </ng-container>
-  </ion-button>
+    <ngx-datatable-footer style="position: fixed; bottom: 0px;" *ngIf="true">
```

Comment:
Try to put CSS code into SCSS file, use class here instead.

What is `*ngIf="true"` for ?

Reply:
need to remove those styles and ngIf.